build.js

```diff
@@ -64,16 +64,78 @@ function getOutputPath(pkg) {
 	return `./web/js/dist/${pkgName}.min.js`;
 }
 
+/** @type {lightningcss.Visitor} */
```

Comment:
No use without TypeScript?

Reply:
JSDocs are useful for autocomplete still, helpful especially when there's a lot of properties, like in this case.